test/__snapshots__/background.js.snap

```diff
@@ -0,0 +1,41 @@
+// Vitest Snapshot v1
+
+exports[`bg attachments 1`] = `
+"/* layer: default */
+.bg-fixed{background-attachment:fixed;}
+.bg-local{background-attachment:local;}
+.bg-scroll{background-attachment:scroll;}"
+`;
+
+exports[`bg origins 1`] = `
+"/* layer: default */
+.bg-origin-border{background-origin:border-box;}
+.bg-origin-padding{background-origin:padding-box;}
+.bg-origin-content{background-origin:content-box;}"
+`;
+
+exports[`bg positions 1`] = `
+"/* layer: default */
+.bg-bottom{background-position:bottom;}
+.bg-center{background-position:center;}
+.bg-left{background-position:left;}
+.bg-right{background-position:right;}
+.bg-top{background-position:top;}"
```

Comment:
It seems that we do not support "nested" positions like `bg-left-bottom`, `bg-left-top`, `bg-right-bottom`, `bg-right-top`. Is it intentional?

Reply:
@AnnaRybkina - good catch, fixed!